src/containers/Application/Part1.js

```diff
@@ -1,6 +1,37 @@
-import React from 'react'
+import React, { useState } from 'react'
+import BasicInfo from '../../components/ApplicationForm/BasicInfo'
 
 // form part 1
-export default ({ children }) => {
-  return <div>{children}</div>
+export default () => {
+  const [states, setStates] = useState({
+    firstName: '',
+    lastName: '',
+    gender: '',
+    ethnicity: {
+      asian: false,
+      black: false,
+      caucasian: false,
+      hispanic: false,
+      middleEastern: false,
+      nativeHawaiian: false,
+      northAmerica: false,
+      other: false,
+      preferNot: false,
+    },
+    isOfLegalAge: null,
+    phoneNumber: '',
+    school: '',
+    major: '',
+    educationLevel: '',
+    graduation: 0,
+    hackathonsAttended: 0,
+    contributionRole: '',
+    location: '',
```

Comment:
Do we need to declare all these as empty? Can we init it as an empty object

Reply:
Might be better to replace this with `updateHackerApplication` once its pushed to master

Reply:
I think initializing this is fine - we need a way to set the initial values if it's a new application